app/template/[templateId]/history/page.tsx

```diff
@@ -0,0 +1,105 @@
+
+    return (
+        <PageWrapper title={"Template History"}>
+            <BackButton />
+            {loading && <p>Template history is loading...</p>}
+            <div>
+                {lastPublication && (
+                    <>
+                        <h1 className="with-subheader">{lastPublication?.name || 'Unknown'}</h1>
+                        <div className="subheader">
+                            <div data-testid="author">{`by ${lastPublication?.versionedBy?.affiliation?.name}`}</div>
+                            <div>
+                                <span data-testid="latest-version" className={styles.historyVersion}>Version {lastPublication?.version.slice(1)}</span>
+                                <span data-testid="publication-date">Published: {lastPublicationDate}</span>
+                            </div>
+                        </div>
+                    </>
+                )}
+
+                <h2>History</h2>
+                <table>
+                    <thead>
+                        <tr>
+                            <th>Action</th>
+                            <th>User</th>
+                            <th>Time and Date</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {
+                            sortedTemplates.length > 0
+                                ? sortedTemplates.map((item, index) => {
+                                    const publishDate = formatWithTimeAndDate(item?.created);
+                                    const versionedBy = item?.versionedBy;
+
+                                    return (
+                                        <tr key={`${item?.id}-${index}`}>
+                                            <td>
+                                                <div>Published {item?.version}</div>
+                                                <div>
+                                                    <small className={styles.changeLog}>
+                                                        Change log:<br />{item?.comment}
+                                                    </small>
+                                                </div>
+                                            </td>
+                                            <td>
+                                                {versionedBy
+                                                    ? `${versionedBy.givenName || ''} ${versionedBy.surName || ''}`
+                                                    : 'Unknown'}</td>
+                                            <td>{publishDate}</td>
+                                        </tr>
+                                    );
+                                })
+                                : <tr><td colSpan={3}>No template history available.</td></tr>
+
+                        }
+
+                    </tbody>
+                </table>
```

Comment:
Should we maybe be using the react-aria table for this instead?
See: https://react-spectrum.adobe.com/react-aria/Table.html

Reply:
@andrewebdev - Yes thanks. I converted it to use React Aria Table, and also added it to the styleguide.

One weird thing is that the <Cell> component doesn't include "colspan". Sometimes these types of libraries have some weird quirks.

Reply:
We should also copy the styles from their components. @fraserclark and I have been doing that. We just copy the CSS from their examples, you just expand the css bits to get to it. We can later customize the styling from there, but it's a good starting point.

Reply:
Thanks @andrewebdev. I copied the styles over to `_elements.scss` and applied the styles. I retained the table styles I added as well, because the css from React Aria Table wasn't enough to simulate the layout in the wireframe.